### ISSUE TITLE
1.23 gradient descent equation rendering

### ISSUE THREAD

**Issue by bmbigbang** [udacity/dlnd-issue-reports#429]
There seems to be a problem with rendering some of the equations from TeX on the webpage. I am using Chrome version  63.0.3239.132

**Comment by bmbigbang** [comment]
I am from the December cohort

**Comment by lcrucks** [comment]
lesson 2.1.23
![latex.JPG](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/588b9ba628c7741a00ff4082/125516c66e82c728ace21e0d46e4958e269ac1e5cdf9ad4cae58f3806b1c6526715cfb2d24dcba29be1123530a591aac4059094cf5fc8d39baa47e21d40854b088270a36d43703adbf881eba00bc21894887bfbe824ec0fa70.JPG)

**Comment by lcrucks** [comment]
From another student: "Much of the latex formulas on the "Lesson 1: Introduction to Neural Networks" page 23 Gradient Descent do not display correctly which makes it difficult to follow along. Attached is a screenshot."  ![latex_errors.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/588b9ba628c7741a00ff4082/125516c66e82c728ace21e0d46e4958e269abcac9dfbb051b71fbcd6395428792c07b7692ac7b670e71a7d5556561ffd42505d11faad892aa7e034798f5819f1c32f0e2a92624cf5f2d75dad0ba8218c4584b3bd8c4dc3ea3426e588864b2c53.png)

**Comment by bmbigbang** [comment]
This was fixed almost as quickly as I reported it so perhaps its a cache issue if you are still facing it.
